modules/ROOT/pages/6.2-release-notes.adoc

```diff
@@ -0,0 +1,536 @@
+
+This regression was fixed in {productname} 6.2 and pressing *Enter* or *Return* in this circumstance no longer incorrectly removes these elements.
+
+=== Fixed a regression where pressing Enter deleted selected `contenteditable="false" <pre>` elements
+
+A regression found in {productname} 6.0 caused `preformatted` elements with `contenteditable="false"` attributes to incorrectly insert a newline when *Enter* or *Return* were pressed.
+
+This regression was fixed in {productname} 6.2 and pressing *Enter* or *Return* no longer inserts an erroneous newline.
+
+
+=== The `editor.insertContent()` API did not respect the `no_events` argument
+
+If, for example, `editor.insertContent('<p>test</p>', { no_events: true });` was run, `BeforeGetContent` and `SetContent` events were still fired, despite `no_events` being set to `true`.
+
+With this update, the API respects the `no_events: true` setting. Adding content to a {productname} document with `no_events` set to true now adds the content without firing events, as expected.
+
+[[deprecated]]
+== Deprecated
+
+=== The `autocompleter` configuration property, `ch`, has been deprecated
+
+The `ch` autocompleter configuration property will be removed in the next major release.
+
+Use the `trigger` property instead.
+
+[source,js]
+----
+trigger: '<string>',
+----
+
+
+[[known-issues]]
+== Known issues
+
+This section describes issues that users of {productname} 6.2 may encounter and possible workarounds for these issues.
+
+There are three known issues in {productname} 6.2, all affecting the new premium plugin, xref:mergetags.adoc[Merge Tags].
+
+=== Merge Tags cannot be manually added to a {productname} document in some circumstances
+
+There are several circumstances where typing the Merge tags prefix (by default, `+{{+`) does not activate the `autocompleter`. Consequently, choosing a Merge tag from the expected pop-up menu is not possible because the pop-up menu does not present.
+
+Moreover, in the circumstances where the `autocompleter` does not activate, even typing the entire Merge tag does not result in the string being turned in to a Merge tag.
+
+There is a workaround, however.
+
+If typing the Merge tags prefix does not work as expected, clicking the Merge tags toolbar button and selecting the desired Merge tag from the resultant menu inserts a functioning Merge tag.
+
+=== Merge tags does not validate inserted strings that match merge tag syntax
+
+If a string is pasted or otherwise inserted into a {productname} document and that string presents as a complete merge tag, the Merge tag plugin will turn the string into an apparently working merge tag, regardless of the string’s content.
+
+For example, if a {productname} instance is running the Merge tags plugin and the plugin is using the default prefix and suffix characters — +{{+` and +}}+` — pasting
```

Comment:
```suggestion
For example, if a {productname} instance is running the Merge tags plugin and the plugin is using the default prefix and suffix characters — `+{{+` and `+}}+` — pasting
```
Rendering incorrectly due to the missing backticks.

Reply:
(discussion): accept suggestion